app/assets/stylesheets/src/shared/_scaffolds.scss

```diff
@@ -238,6 +238,13 @@ picture {
   stroke-width: 1.25;
 }
 
+.avatar-64px {
+  --tblr-avatar-size: 4rem;
+  --tblr-avatar-font-size: 2rem;
+  --tblr-avatar-status-size: 1rem;
+  --tblr-avatar-icon-size: 2.5rem;
+}
```

Comment:
I'm not sure where best to put this class

Reply:
It's also not very clear which name is better
It is between the default `lg` and `xl`.

Reply:
chatgpt suggests to have name like lg/xl to call size between lg and xl. I think, it's Ok. (avatar-lg/xl or avatar-lg-xl or avatar-lxl)
In hub we have case when added the custom site with suffix `-cs`
It also possible to define the actual sixe, but in such case I's prefer to have rems in names, since they are used in variables.

Reply:
Changed to `avatar-md-custom`

Reply:
Why md if it between lg and xl? maybe avatar-lg-custom?

Reply:
@ksnitsky what do you think about name? @wrmk maybe you also have suggestions?

Reply:
Since old md size in tabler was 64x64

Reply:
i like the old one, `avatar-64px`, i expect that it will display image with 64*64

`avatar-md-custom` or smth like this unclear, even if I look at the implementation, some magic variables, unclear in what cases I should reuse this class